packages/bpk-tokens/gulpfile.babel.js

```diff
@@ -24,32 +25,42 @@ import gulpTheo from 'gulp-theo';
 import { flatten } from 'lodash';
 import gulpMerge from 'gulp-merge';
 import jsonLint from 'gulp-jsonlint';
+import adjustTypography from './formatters/adjust-typography';
 
 import bpkScss from './formatters/bpk.scss';
 import bpkEs6Js from './formatters/bpk.es6.js';
 import bpkIosJson from './formatters/bpk.ios.json';
-import bpkRawJson from './formatters/bpk.raw.json';
+import bpkRawJson, {
+  bpkRawJsonAndroid,
+  bpkRawJsonIos,
+} from './formatters/bpk.raw.json';
 import bpkCommonJs from './formatters/bpk.common.js';
 import bpkAndroid from './formatters/bpk.android.xml';
 import bpkDefaultScss from './formatters/bpk.default.scss';
-import bpkReactNativeEs6Js from './formatters/bpk.react.native.es6.js';
-import bpkReactNativeCommonJs from './formatters/bpk.react.native.common.js';
+import {
+  bpkReactNativeEs6JsIos,
+  bpkReactNativeEs6JsAndroid,
+} from './formatters/bpk.react.native.es6.js';
+import {
+  bpkReactNativeCommonJsIos,
+  bpkReactNativeCommonJsAndroid,
+} from './formatters/bpk.react.native.common.js';
 
 const PLATFORM_FORMATS = {
   web: ['scss', 'default.scss', 'raw.json', 'common.js', 'es6.js'],
   ios: [
     'ios.json',
     'raw.ios.json',
     'react.native.ios.js',
-    { format: 'react.native.es6.js', nest: true },
-    { format: 'react.native.common.js', nest: true },
+    { format: 'IOS_react.native.es6.js', nest: true },
+    { format: 'IOS_react.native.common.js', nest: true },
   ],
   android: [
     'android.xml',
     'raw.android.json',
     'react.native.android.js',
-    { format: 'react.native.es6.js', nest: true },
-    { format: 'react.native.common.js', nest: true },
+    { format: 'ANDROID_react.native.es6.js', nest: true },
```

Comment:
Can you explain these two changes a bit? I.e. the prefixes

Reply:
Basically, I needed a way to register different variants of the `react.native.es6.js` and `react.native.common.js` formatters to `gulp-theo`. The only way I could see to do this was to add something to the identifiers to differentiate them.

I know this solution ain't pretty, but theo seems pretty closed in the way it invokes the formatters. If there's a better way I'm all ears 👍